### PR TITLE
Simple status page & flower fixes

### LINKED ISSUE #309: simple health status page

### PR DESCRIPTION
**Pull request by ckcollab**
# @ mention of reviewers
@jimmykodes 

# A brief description of the purpose of the changes contained in this PR.
Adds simple status page, showing most recent 250 submissions and links to rabbit/flower debugging tools. Also, merging PR #318 into this: _"Previously Flower/RabbitMQ management would not have very many useful details for Rabbit jobs. We couldn't see compute workers or which jobs they were picking up. This PR should fix that."_

Resolves #309 and #317 

# A checklist for hand testing
- [x] recent submissions appear in submission list; none > 2 days old and not > 250
- [x] rabbit/flower links on the bottom of the page work
- [x] new admin menu (top right) should work
- [x] go to host:5555 and login with flower basic auth to get details about flower while tasks are running, each compute worker should be listed and the jobs they're executing should be apparent

For ref, new menu:
image

# Checklist
- [x] Code review by me 
- [x] Hand tested by me 
- [x] I'm proud of my work
- [x] Code review by reviewer
- [x] Hand tested by reviewer
- [x] Ready to merge

### CODE REVIEW

File: src/static/riot/admin/server_status.tag [hunk 1]
```diff
@@ -1,39 +1,6 @@
+<!-- Old unused tag, we could pull this back at some point? had a gauge and such -->
```
Comment (jimmykodes) [inline src/static/riot/admin/server_status.tag]: maybe just pull this tag file altogether? can always go find it again in git.

Comment (ckcollab) [reply src/static/riot/admin/server_status.tag]: pulled! ✔️

File: src/templates/pages/server_status.html [hunk 2]
```diff
@@ -2,8 +2,70 @@
...
+        <div id="external_monitors" class="ui two column grid">
+            <div class="column">
+                <div class="ui fluid card">
+                    <a class="image" href="#">
+                        <img class="ui large image" src="/static/img/RabbitMQ.png">
+                    </a>
+                    <div class="content">
+                        <a class="header" href="//{{RABBITMQ_MANAGEMENT_URL}}" target="_blank">RabbitMQ</a>
+                        <div class="meta">
+                            <a href="#">
```
Comment (jimmykodes) [inline src/templates/pages/server_status.html]: Should this be an `a` tag? or should it have an `href`?

File: src/templates/pages/server_status.html [hunk 3]
```diff
@@ -2,8 +2,70 @@
...
+                </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+
+        <h1>Monitor queues</h1>
+        <div id="external_monitors" class="ui two column grid">
+            <div class="column">
+                <div class="ui fluid card">
+                    <a class="image" href="#">
```
Comment (jimmykodes) [inline src/templates/pages/server_status.html]: same here?

File: src/templates/pages/server_status.html [hunk 4]
```diff
@@ -2,8 +2,70 @@
...
+                                exchanges, and other administrative features relating to RabbitMQ e.g. Creating users,
+                                adding v-hosts, and creating policies.
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="column">
+                <div class="ui fluid card">
+                    <a class="image" href="#">
```
Comment (jimmykodes) [inline src/templates/pages/server_status.html]: and here

File: src/templates/pages/server_status.html [hunk 5]
```diff
@@ -2,8 +2,70 @@
...
+                        <img class="ui large image" src="/static/img/Flower.png">
+                    </a>
+                    <div class="content">
+                        <a class="header" href="//{{ FLOWER_URL }}" target="_blank">Flower</a>
+                        <div class="meta">
+                            <a href="#">
```
Comment (jimmykodes) [inline src/templates/pages/server_status.html]: and here

File: src/templates/pages/server_status.html [hunk 6]
```diff
@@ -2,8 +2,70 @@
...
+            <thead>
+            <th>Competition</th>
+            <th>Submitter</th>
+            <th>Submitted at</th>
+            <th>Status</th>
+            </thead>
+            <tbody>
+                {% for sub in submissions %}
+                <tr>
+                    <td>{{ sub.phase.competition }}</td>
```
Comment (jimmykodes) [inline src/templates/pages/server_status.html]: maybe also add submission pk to this list? so we could go find the sub in admin/shell plus or something and get task id and send kill signal if we need to?

Comment (ckcollab) [reply src/templates/pages/server_status.html]: ✔️

File: src/templates/pages/server_status.html (hunk 3 above)
Comment (ckcollab) [reply src/templates/pages/server_status.html]: ✔️

File: src/templates/pages/server_status.html (hunk 2 above)
Comment (ckcollab) [reply src/templates/pages/server_status.html]: Oh.. didn't even notice these, I just removed them? ✔️

File: src/templates/pages/server_status.html (hunk 4 above)
Comment (ckcollab) [reply src/templates/pages/server_status.html]: ✔️

File: src/templates/pages/server_status.html (hunk 5 above)
Comment (ckcollab) [reply src/templates/pages/server_status.html]: _(pulled as well)_ ✔️

**Pull request merged**